README.md

```diff
@@ -1,15 +1,50 @@
 # Mink WebDriver Classic Driver
 
+[![PHP Version](http://poser.pugx.org/mink/webdriver-classic-driver/require/php)](https://packagist.org/packages/mink/webdriver-classic-driver)
 [![Latest Stable Version](https://poser.pugx.org/mink/webdriver-classic-driver/v)](https://packagist.org/packages/mink/webdriver-classic-driver)
 [![Latest Unstable Version](https://poser.pugx.org/mink/webdriver-classic-driver/v/unstable)](https://packagist.org/packages/mink/webdriver-classic-driver)
 [![Total Downloads](https://poser.pugx.org/mink/webdriver-classic-driver/downloads)](https://packagist.org/packages/mink/webdriver-classic-driver)
 [![CI](https://github.com/minkphp/webdriver-classic-driver/actions/workflows/ci.yml/badge.svg)](https://github.com/minkphp/webdriver-classic-driver/actions/workflows/ci.yml)
 [![License](https://poser.pugx.org/mink/webdriver-classic-driver/license)](https://github.com/minkphp/webdriver-classic-driver/blob/main/LICENSE)
 [![codecov](https://codecov.io/gh/minkphp/webdriver-classic-driver/branch/main/graph/badge.svg?token=11hgqXqod9)](https://codecov.io/gh/minkphp/webdriver-classic-driver)
 
-## Usage Example
+A [php-webdriver](https://github.com/php-webdriver/php-webdriver)-based driver
+for [Mink](https://github.com/minkphp/Mink), supporting Selenium 2-4 and tested on Chrome, Firefox and Edge.
 
-``` php
+## Installation
+
+### For Mink/Standalone Usage
+
+Simply install the package with composer and check the [Standalone Usage Example](#standalone-usage-example) below.
+
+```shell
+composer require "behat/mink" "mink/webdriver-classic-driver"
+```
+
+### For Behat/BDD Usage
+
+1. Add all the relevant packages - typically this will be for testing your application, so you'd want to use `--dev`.
+    ```shell
+    composer require --dev "behat/behat" "behat/mink" "friends-of-behat/mink-extension" "mink/webdriver-classic-driver"
+    ```
+2. Configure Behat to enable the Mink Extension and set it to use this driver, for example:
+    ```yaml
+    # behat.yml
+    default:
+        extensions:
+            Behat\MinkExtension\Extension:
+                browser_name: chrome
+                base_url: https://example.com/
+                sessions:
+                    default:
+                        webdriver-classic: ~
```

Comment:
I don't think this readme should document the configuration of the Behat MinkExtension (this is something documented in the MinkExtension documentation instead).

Reply:
I would normally agree, but then the `webdriver-classic` key would not be documented anywhere.

Reply:
Do you know how this problem is solved in the MinkSelenium2Driver? It's `selenium2` key should be advertised somewhere.

Reply:
There's one here: https://docs.behat.org/en/v2.5/cookbook/behat_and_mink.html#:~:text=The%20easiest%20way%20to%20get%20started%20is%20to%20go%20with%20goutte%20and%20selenium2%20drivers%2C%20but%20note%20that%20there%E2%80%99s%20bunch%20of%20other%20drivers%20available%20for%20Mink%20%2D%20read%20about%20them%20in%20Mink%20documentation.

There isn't any mention in the mink documentation as far as I could see: https://mink.behat.org/en/latest/drivers/selenium2.html

@stof is also right - that key is not even in this repository; it's actually defined in `FriendsOfBehat/MinkExtension`.

I'd like to keep it out of this repository/readme, but I have no guarantees it will be documented elsewhere.

Reply:
Removed the change.